internal/common/namespace/namespace.go

```diff
@@ -6,7 +6,9 @@ type Namespace = string
 
 const (
 	Organization Namespace = "organization"
+	Group        Namespace = "group"
 	Repository   Namespace = "repository"
+	Project      Namespace = "project"
```

Comment:
It's confusing to have these in the Namespace, because we use Namespace for many purposes, and it'd be really hard to distinct real namespaces from virtual ones.
I'd move the translations directly to the map in the translator function.